lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -200,8 +208,13 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
     ),
     '#title' => t("Caption"),
     '#default_value' => $caption,
+    '#description' => t("This is public facing. Do not enter private user information."),
   );
 
+  if (!user_access('administer modules')) {
```

Comment:
@angaither ah great - changed! 

@namimody this addresses your comment - thanks for flagging too!